Makefile

```diff
@@ -11,6 +11,23 @@ STAGINGSERVER ?= node-www
 
 OSTYPE := $(shell uname -s | tr '[A-Z]' '[a-z]')
 
+ifdef QUICKCHECK
+  QUICKCHECK_ARG := --quickcheck
+endif
+
+ifdef ENABLE_V8_TAP
+  TAP_V8 := --junitout v8-tap.xml
+  TAP_V8_INTL := --junitout v8-intl-tap.xml
+  TAP_V8_BENCHMARKS := --junitout v8-benchmarks-tap.xml
+endif
+
+ifdef DISABLE_V8_I18N
+  V8_TEST_NO_I18N := --noi18n
+  V8_BUILD_NO_I18N := i18nsupport=off
```

Comment:
Suggestion: rename this to `V8_BUILD_OPTIONS` and append with `V8_BUILD_OPTIONS += i18nsupport=off`, then I as a user can add flags of my own.